components/calendar/src/calendar.js

```diff
@@ -0,0 +1,368 @@
+import { colors, spacers } from '@dhis2/ui-constants'
+import { IconChevronLeft16, IconChevronRight16 } from '@dhis2/ui-icons'
+import cx from 'classnames'
+import { useDatePicker } from 'multi-calendar-dates'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+
+export const Calendar = ({
+    onDateSelect,
+    calendar,
+    initialDate,
+    dir = 'ltr',
+    locale = 'en',
```

Comment:
would it be worth it to specify a default non-North American variant of English (e.g. `en-GB`, `en-NG`, etc.,) so that our dates e.g. if displayed when `showSelectedDate=true` are formatted in DD MM YYYY rather than the US/Canada MM DD YYYY format?